schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY, 
```

Comment:
- [OPTIONAL] I think it can be a good idea if you mark the id column as the primary key `PRIMARY KEY (id)` 😁 😁